drivers/mrf24j40/mrf24j40_netdev.c

```diff
@@ -106,6 +106,8 @@ static int _send(netdev_t *netdev, const struct iovec *vector, unsigned count)
         len = mrf24j40_tx_load(dev, ptr->iov_base, ptr->iov_len, len);
         if (i == 0) {
             dev->header_len = len;
+            /* Grab the FCF bits from the frame header */
+            dev->fcf_low = *(uint8_t*)(ptr->iov_base);
```

Comment:
I think this (and the above header_len assignment) fits better in the mrf24j40_tx_load function, or is there a hidden reason why it shouldn't be there?

Reply:
Not sure about this one, I like to keep the `tx_load` function clean (radio frame loading only) and not add additional logic to it to check the content of the frame.